go.mod

```diff
@@ -1,4 +1,4 @@
-module github.com/milkyway-labs/milkyway/v2
+module github.com/milkyway-labs/milkyway/v3
```

Comment:
Can you add a Changeset entry log? It's something that I've added recently. You can read how to do it inside the `.changeset` folder of the repository. It should be pretty straightforwarded but if you have any issue let me know!

Reply:
I added a changeset entry, please check it.